identity-vault/getting-started/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "getting-started",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build && cap sync",
+    "preview": "vite preview",
+    "test.e2e": "cypress run",
+    "test.unit": "vitest",
+    "lint": "eslint"
+  },
+  "dependencies": {
+    "@capacitor/android": "5.7.0",
+    "@capacitor/app": "5.0.7",
+    "@capacitor/core": "5.7.0",
+    "@capacitor/haptics": "5.0.7",
+    "@capacitor/ios": "5.7.0",
+    "@capacitor/keyboard": "5.0.8",
+    "@capacitor/status-bar": "5.0.7",
+    "@ionic-enterprise/identity-vault": "^5.12.4",
+    "@ionic/react": "^7.0.0",
+    "@ionic/react-router": "^7.0.0",
+    "@types/react-router": "^5.1.20",
+    "@types/react-router-dom": "^5.3.3",
+    "ionicons": "^7.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router": "^5.3.4",
+    "react-router-dom": "^5.3.4"
+  },
+  "devDependencies": {
+    "@capacitor/cli": "5.7.0",
+    "@testing-library/dom": ">=7.21.4",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/react": "^18.0.27",
+    "@types/react-dom": "^18.0.10",
+    "@vitejs/plugin-legacy": "^5.0.0",
+    "@vitejs/plugin-react": "^4.0.1",
+    "cypress": "^13.5.0",
+    "eslint": "^8.35.0",
+    "eslint-plugin-react": "^7.32.2",
+    "jsdom": "^22.1.0",
+    "terser": "^5.4.0",
+    "typescript": "^5.1.6",
+    "vite": "^5.0.0",
+    "vitest": "^0.34.6"
+  },
```

Comment:
Align non-Ionic dependencies/versions with other projects in the monorepo.

Reply:
Do we need to do a `pnpm outdated -r` and `pnpm update -r <select items here>` to bring some of this in line?

That is the direction I have been taking with the Angular and Vue projects. That is, when I create a new project I take the opportunity to update as many deps as I can across all projects.

Reply:
Just to make sure, you're referring to `cypress`, `eslint`, `typescript, `vite`, `vitest`, etc. dev dependencies, right?

Reply:
@jon-ionic - Correct, anything that isn't on the `@ionic`, `@ionic-enterprise`, or `@capacitor` scope. 
 
@kensodemann - Is `pnpm update -r --latest` intelligent enough to only update packages that use the dependency? For example, would `pnpm update -r --latest @ionic-enterprise/identity-vault` add IV to projects that don't have it listed in `package.json`?

Reply:
@eric-horodyski: `pnpm upgrade` does not `add` only `update`. As such if you run `pnpm update -r --latest foo` it will only update `foo` in packages that actually have `foo` as a dependency.

Reply:
As an aside, maybe we should have a top-level npm script that does the following:

```
pnpm update -r --latest @ionic-enterprise/* @ionic/* @capacitor/*
```